app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/MultipleDisposalsTriageController.scala

```diff
@@ -953,6 +958,13 @@ class MultipleDisposalsTriageController @Inject() (
       }
     }
 
+  private def getTaxYearByTaxYearExchanged(taxYearExhanged: Option[TaxYearExchanged]): Int =
+    taxYearExhanged match {
+      case Some(TaxYearExchanged.TaxYear2020) => 2020
+      case Some(TaxYearExchanged.TaxYear2021) => 2021
+      case _                                  => 0
```

Comment:
Should this be None instead of zero?

Reply:
Or is used somewhere as an Int?

Reply:
Yeah Good idea, I will wrap the written value with Option.